packages/developer-portal/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/form-schema/form-fields.ts

```diff
@@ -12,6 +12,8 @@ export type FieldKeys =
   | 'noRegField'
   | 'aboutField'
   | 'iconImageUrlField'
+  | 'buildingNameField'
+  | 'buildingNumberField'
```

Comment:
Please add these new fields to the intialValues object of OrganisationForm in:
`components/pages/settings/developer-settings-organisation-tab/organisation-form/organisation-form.tsx`